.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+#     strategy:
+#       matrix:
+#         python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 mkdocs
```

Comment:
Can you add `mkdocs-material` to the pip install

https://pypi.org/project/mkdocs-material/

Reply:
Nevermind! Apparently I can do that